agent_shared/src/config/mod.rs

```diff
@@ -1198,4 +1311,87 @@ mod tests {
             },
         );
     }
+
+    #[test]
+    #[serial]
+    fn test_public_url_defaults_to_application_url() {
+        temp_env::with_vars(
+            [
+                ("UNICORE__EVENT_STORE__CONNECTION_STRING", Some("postgresql://:test:")),
+                ("UNICORE__SECRET_MANAGER__STRONGHOLD_PASSWORD", Some("unsafe-password")),
```

Comment:
Are those env variables required in the test, even if `ApplicationProfile::Development` is used?

Reply:
indeed ==> removed :wastebasket: